Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM continuumio/miniconda3:4.9.2
 
 ENV PATH /opt/conda/bin:$PATH
 ENV PYTHONUNBUFFERED TRUE
-RUN conda install mamba -c conda-forge && conda clean --all
+RUN conda install mamba=0.14.0 tini=0.19.0 -c conda-forge && conda clean --all
```

Comment:
Just out of curiosity, why do we need to pin the version of `mamba`? Very cool to use `tini` here!

Reply:
Yeah, good question @dgergel. I'm going to try controlling `mamba`. My thinking is that we're controlling the version of `minicona` (up in line 1 or 2 of this file), so we should pin `mamba`, also.

Reply:
Ahh okay. Good to know. It's a shame we can't pin version 5 - the semantic versioning of mamba should have really considered the value of mamba #5